pyproject.toml

```diff
@@ -121,6 +121,7 @@ docu = [
     "sphinxcontrib-mermaid",
     "pylint",
     "livereload",
+    "tabulate==0.9.0",
```

Comment:
Is the installation of tabulate version 0.9.0 required, or may higher versions also work? Are there any troubles in installation, if the version number remains undefined? only specify version if required.